sdk/eventhub/Azure.Messaging.EventHubs/samples/Azure.Messaging.EventHubs.Samples.csproj

```diff
@@ -17,4 +17,7 @@
   
   <!-- Import the references to the Azure client libraries -->
   <Import Project="$(MSBuildThisFileDirectory)..\Azure.Client.Library.References.props" />
+
```

Comment:
This file appears to be very out of date.   These changes would cause issues and seem orthogonal to the stated purpose of the PR.   Please revert.